ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -30,9 +30,9 @@ public async Task PostDraft_ReturnsPageWithAllModelErrors_IfModelNotValid()
                 ["ClinicalDetails.BCGVaccinationState"] = "Yes",

```

Comment:
Were no changes to the tests required? I think we're already covering similar cases in here anyway, and don't check every single date as the validation is identical for each, so it's probably fine

Reply:
Yep apart from naming changes nothing needed to be added without retesting the same kind of logic